warehouse/router_test.go

```diff
@@ -906,6 +906,12 @@ func TestRouter(t *testing.T) {
 				pgResource, err := resource.SetupPostgres(pool, t)
 				require.NoError(t, err)
 
+				config.Set("DB.host", pgResource.Host)
+				config.Set("DB.port", pgResource.Port)
+				config.Set("DB.user", pgResource.User)
+				config.Set("DB.password", pgResource.Password)
+				config.Set("DB.name", pgResource.Database)
+
```

Comment:
using the global config and parallel tests seems like a recipe for disaster to me

Reply:
yeah
only option was to remove the `t.Parallel()`.
can't use `t.Setenv` in parallel tests.

Reply:
does this test rely on these config options?

Reply:
correction: it doesn't, removing these lines